doc/ssl/SSL_CONF_cmd.pod

```diff
@@ -227,6 +232,11 @@ Attempts to use the file B<value> as the set of temporary DH parameters for
 the appropriate context. This option is only supported if certificate
 operations are permitted.
 
+=item B<NoRenegotiation>
+
+Disables all attempts at renegotiation in TLSv1.2 and earlier, same as setting
+B<SSL_OP_NO_RENEGOTIATION>.
```

Comment:
The grammar here (and above) is a little strange, but we can probably let it slide.

Reply:
It is as it appears in the master version